plotly_resampler/figure_resampler/figure_resampler_interface.py

```diff
@@ -892,13 +942,7 @@ def add_trace(
         hf_hovertext: Iterable, optional
             The original high frequency hovertext. If set, this has priority over the
             trace its ```hovertext`` argument.
-        check_nans: boolean, optional
-            If set to True, the trace's data will be checked for NaNs - which will be
-            removed. By default True.
-            As this is a costly operation, it is recommended to set this parameter to
-            False if you are sure that your data does not contain NaNs (or when the
-            downsampler can handle NaNs, e.g., EveryNthPoint). This should considerably
-            speed up the graph construction time.
+        TODO docs
```

Comment:
Why did you remove `check_nans` documentation?

Reply:
Whoops, this was a small mistake, due to the merge conflict with `main` 🙈